docs/IntegrationWithExistingApps.md

```diff
@@ -157,9 +157,11 @@ Go to the root directory for your project and create a new `package.json` file w
 Next, you will install the `react` and `react-native` packages. Open a terminal or command prompt, then navigate to the root directory for your project and type the following commands:
 
 ```
-$ npm install --save react@16.0.0-alpha.12 react-native
+$ npm install --save react@16.0.0-beta.5 react-native
```

Comment:
@hramos when i use `react-native init` to create a new project, `alpha.12` is the version that actually get installed.

Reply:
Gotcha, it's because 0.48 (the latest stable) is still on alpha.12: https://github.com/facebook/react-native/blob/0.48-stable/package.json

I'll need to update the wording here.

Reply:
Using beta.5 here is fine, because these are the docs for master and won't be displayed in the current 0.48 docs. We'd need to edit the markdown file in the 0.48 branch directly in this case.

Reply:
Gotcha, it make sense.

Reply:
Fixed in 6488842e1a02501c34c4be0454e6df15009090a3 (0.48, current stable) and 031feb694257d6ffb8aa8fd7b4019d96dac7155a (0.49, next RC).

Reply:
Thanks for figuring this out BTW.